src/app/components/contacts-add/contacts-add.component.html

```diff
@@ -0,0 +1,57 @@
+<!-- Begin Page Content -->
+<br />
+<h1 class="h3 mb-4 text-gray-800">{{ pageTitle }}</h1>
+
+<!-- /.container-fluid -->
+
+<div class="card shadow mb-4">
+    <div class="card-body">
+        <form class="user" [formGroup]="addNewContactForm" (ngSubmit)="onSubmit()">
+            <div class="form-group row">
+                <div class="col-sm-4 mb-3 mb-sm-0">
+                    <input type="text" class="form-control" formControlName="name" placeholder="Name">
+                    <div *ngIf="submitted && f.name.errors" class="invalid-feedback">
+                        <div *ngIf="f.name.errors" class="text-danger">
+                            This field is mandatory</div>
+                    </div>
+                </div>
+                <div class="col-sm-4">
+                    <input type="email" class="form-control" formControlName="email" placeholder="Email">
+                    <div *ngIf="submitted && f.email.errors" class="invalid-feedback">
+                        <div *ngIf="f.email.errors" class="text-danger">
+                            This field is mandatory</div>
+                    </div>
+                </div>
+                <div class="col-sm-4">
+                    <input type="text" class="form-control" formControlName="phone" placeholder="phone">
+                    <p><small>you can add more than on EX: 012234567, 032278999, 8765544321 </small></p>
```

Comment:
It would be better to have an input field for each extra phone number, send phone as array of phone numbers.